### PR TITLE
Add function content and test

### PR DESCRIPTION
**Pull request by scarlqq**

### CODE REVIEW

File: src/outliers/visualize_outliers.py [hunk 1]
```diff
@@ -1,16 +1,29 @@
-def visualize_outliers(dataframe, columns=None):
+import pandas as pd
+import altair as alt
+
+def visualize_outliers(dataframe, columns=None, type='violin'):
     """
-    A function that plot out the given data with outliers marked.
+    A function that plot the distribution of the give data.
```
Comment (karanpreetkaur) [inline src/outliers/visualize_outliers.py:6]:
```suggestion
    A function that plot the distribution of the given data.
```

**Review by karanpreetkaur — CHANGES_REQUESTED** [review]
This looks good. Thanks @scarlqq 

Just a small correction in docstring.

File: src/outliers/visualize_outliers.py (hunk 1 above)
Comment (scarlqq) [reply src/outliers/visualize_outliers.py:6]: Thank you for all the corrections there are really tooooooo much typo for me

**Comment by lipcai** [comment]
looks great

**Pull request merged**